src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiProductController.cs

```diff
@@ -131,12 +130,12 @@ private PriceViewModel GetPriceViewModel(decimal priceExclTax, decimal priceIncl
             };
         }
 
-        private MoneyViewModel GetMoneyViewModel(decimal price, Currency currency)
+        private MoneyViewModel GetMoneyViewModel(decimal price, string currency)
         {
             return new MoneyViewModel()
             {
                 Value = price,
-                DisplayValue = new Money(price, currency).ToString()
+                DisplayValue = new ApiMoney(price, currency).ToString()
```

Comment:
What's the reason for keeping `currency` as variable name here and not in the previous cases?
Let's have it consistent!

Reply:
This is in a separate method, but I agree with you 👍